tests/rustdoc-ui/doctest/test-compile-fail3.rs

```diff
@@ -1,3 +1,4 @@
 //@ compile-flags:--test
 
 "fail
+//~^ ERROR
```

Comment:
(could also switch to `//~v ERROR` to prevent the stderr from changing but it doesn't really matter)